waiter/test/waiter/auth/jwt_test.clj

```diff
@@ -851,6 +761,12 @@
               authenticator (->JwtAuthenticator
                               allow-bearer-auth-api? true false auth-server jwt-validator "password")
               jwt-handler (wrap-auth-handler authenticator handler)]
+          (is (= {:body ::standard-request}
+                 (jwt-handler {:headers {"authorization" "Bearer abcdef"}
+                               :source ::standard-request})))
+          (is (= {:body ::standard-request}
```

Comment:
Isn't this assertion identical to the previous one?

Reply:
My bad, duplicate copy-paste. Removing.

Reply:
Removed the duplicate.